src/IssueViz.Security.UnitTests/OpenInIDE/Api/OpenInIdeStateValidatorTests.cs

```diff
@@ -112,7 +112,7 @@ public void CanHandleOpenInIDERequest_HasPreviousInfoBar_ConnectionIsNowCorrect_
             var infoBar = new Mock<IInfoBar>();
             var infoBarManager = new Mock<IInfoBarManager>();
             infoBarManager
-                .Setup(x => x.AttachInfoBarWithButton(new Guid(HotspotsToolWindow.ToolWindowId), It.IsAny<string>(), It.IsAny<string>(), default))
+                .Setup(x => x.AttachInfoBarWithButton(HotspotsToolWindow.ToolWindowId, It.IsAny<string>(), It.IsAny<string>(), default))
```

Comment:
Miscellaneous: I change the toolwindow to expose a Guid id property since that's what's needed in multiple places.